sigle/src/modules/editor/CoverImage.tsx

```diff
@@ -49,28 +51,44 @@ export const CoverImage = ({ story, setStoryFile }: CoverImageProps) => {
         contentType: blob.type,
       });
 
+      setIsDraggingOver(false);
       setLoadingSave(false);
       setStoryFile({ ...story, coverImage: coverImageUrl });
     },
     [story]
   );
-  const { getRootProps, getInputProps } = useDropzone({
+  const { getRootProps, getInputProps, fileRejections } = useDropzone({
     onDrop,
     accept: 'image/jpeg,image/png',
+    onDragEnter: () => setIsDraggingOver(true),
```

Comment:
could this be replaced by `isDragActive` which is returned by `useDropzone`?

Reply:
ahh yes forgot it's exposing this, will update it now thanks :)